packages/Operators/src/PointCloud/DTK_SplineInterpolationOperator_impl.hpp

```diff
@@ -76,26 +76,41 @@ SplineInterpolationOperator<Basis,DIM>::SplineInterpolationOperator(
     const Teuchos::ParameterList& parameters )
     : Base( domain_map, range_map )
     , d_domain_entity_dim( 0 )
-    , d_range_entity_dim( 0 )      
+    , d_range_entity_dim( 0 )
```

Comment:
what did actually change between these two? don't tell me it's a tab versus space kind of deal...